vue/components/ui/listing-platforme/listing-platforme.vue

```diff
@@ -259,12 +259,59 @@ input[type="text"] {
 }
 
 input[type="text"]:hover {
-    border-color: #dddddd;
+    border-color: $light-white;
 }
 
 input[type="text"]:focus {
-    background-color: #ffffff;
-    border-color: #aaaaaa;
+    background-color: $white;
+    border-color: $light-grey;
+}
+
+body.mobile .listing > .container-platforme,
+body.tablet .listing > .container-platforme {
+    padding: 10px;
+}
+
+body.mobile .listing > .container-platforme > .container-header,
+body.tablet .listing > .container-platforme > .container-header {
+    padding: 0;
+    width: 100%;
+}
+
+body.mobile .listing > .container-platforme > .container-header > .container-header-right ::v-deep,
+body.tablet .listing > .container-platforme > .container-header > .container-header-right ::v-deep {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+body.mobile .listing > .container-platforme > .container-header > .container-header-right ::v-deep > img.logo,

```

Comment:
I don't think you need the `::v-deep`

Reply:
I made tests. and worked only with the v-deep